Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m

```diff
@@ -187,7 +185,6 @@ -(void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
         }
         
         eventDetailViewController.theEvent = event;
-        //eventDetailViewController.title = event.title;
```

Comment:
If you haven't touched the file, leave it. Remove unnecessary comments as you update old code. 

> On Apr 17, 2016, at 3:06 PM, AFrench <notifications@github.com> wrote:
> 
> In Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m:
> 
> > @@ -187,7 +185,6 @@ -(void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
> >          }
> >          
> >          eventDetailViewController.theEvent = event;
> > -        //eventDetailViewController.title = event.title;
> There's a bunch of commented code in the CalendarsViewController.m and a bit in the EventDetailsViewController.m that I haven't touched and wasn't sure if it needed to stay or if I should delete it all.
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
>